lib/components/app/popup.tsx

```diff
@@ -62,19 +62,34 @@ const PopupWrapper = ({ content, hideModal }: Props): JSX.Element | null => {
 
   const title = intl.formatMessage({ id: `config.popups.${id}` })
 
+  /* HACK: Since Bootstrap 3.x does not support adding id or name to navItem, 
+  we have to grab a list of all navItems by className and find the correct button.
+  Since the sliding pane "Leave Feedback" button will always be in the DOM after
+  the navbar button, reverse + find should always find the correct button to return to. */
+
+  const navItemList = Array.from(
+    document.getElementsByClassName('navItem')
```

Comment:
We've done something like this for handling the keyboard up and down arrows in the menu dropdowns, so this should be fine and it seems to work.